tdrs-backend/tdpservice/scheduling/tasks.py

```diff
@@ -41,3 +45,34 @@ def users_to_deactivate(days):
         last_login__gte=datetime.now(tz=timezone.utc) - timedelta(days=days+1),
         account_approval_status=AccountApprovalStatusChoices.APPROVED,
         )
+
+def get_ofa_admin_user_emails():
+    """Return a list of OFA System Admin users."""
+    return [user.email for user in User.objects.filter(
```

Comment:
again, minor performance enhancement, this offloads all of the work that goes into selecting the username from each record to the database (stolen from DataFiles `views.py` so may need adjusting)

```python
recipients = User.objects.filter(
                stt=data_file.stt,
                account_approval_status=AccountApprovalStatusChoices.APPROVED,
                groups=Group.objects.get(name='Data Analyst')
            ).values_list('username', flat=True).distinct()
```